src/main/java/com/endava/soa_5ed/task2/datetime/DateTimeFormatterClass.java

```diff
@@ -0,0 +1,44 @@
+package com.endava.soa_5ed.task2.datetime;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.Locale;
+
+public class DateTimeFormatterClass {
+    private final LocalDateTime dateTime = LocalDateTime.now();
```

Comment:
Initialize the property inside a constructor.

Reply:
Done